tensorflow/python/ops/math_ops.py

```diff
@@ -676,11 +703,11 @@ def angle(input, name=None):
 
   For example:
 
-  ```
-  input = tf.constant([-2.25 + 4.75j, 3.25 + 5.75j], dtype=tf.complex64)
-  tf.math.angle(input).numpy()
-  # ==> array([2.0131705, 1.056345 ], dtype=float32)
-  ```
+  >>> input = tf.constant([-2.25 + 4.75j, 3.25 + 5.75j], dtype=tf.complex64)
```

Comment:
I would recommend using `tf.constant([-1 + j, 2 + 2j], dtype=tf.complex64)` instead as the angles (+/- 45 degrees) are easier to recognize. Plus, then the line won't be extremely long.